### PR TITLE
this is the change to solve the casesand clients not showing in AYs U…

### PR DESCRIPTION
**Pull request by andrewyoungman**
this is the change to solve the case sand clients not showing in AYs UAT ran dashboard task

### CODE REVIEW

File: robot_scripts/PageObjects/dashboard.robot [hunk 1]
```diff
@@ -48,8 +48,8 @@ Click Merits Link When Visible
 Click Merits Case Search Link When Visible
     [Documentation]  Uses: Selenium. Retruns: none.
     ...   Click on the cases and clients link.
-    Wait Until Element Is Visible    css:li[id='CCMS Complex Merits Caseworker']
-
+    #Wait Until Element Is Visible    css:li[id='CCMS Complex Merits Caseworker']
```
Comment (vimalsuresh) [inline robot_scripts/PageObjects/dashboard.robot:51]: can we remove the commented-out lines please?

Comment (andrewyoungman) [reply robot_scripts/PageObjects/dashboard.robot:51]: Yes, shall I do it or would you?

Comment (andrewyoungman) [reply robot_scripts/PageObjects/dashboard.robot]: Done now.

Comment (vimalsuresh) [reply robot_scripts/PageObjects/dashboard.robot]: 👍

**Review by vimalsuresh — APPROVED** [review]

**Pull request merged**